### ISSUE TITLE
Download wget on win7 fails because of TLS 1.2

### ISSUE THREAD

**Issue by 2xyo** [boxcutter/windows#155]
The download of wget on windows 7 fails because https://eternallybored.org/misc/wget/current/wget.exe uses `TLS 1.2` and  .NET Framework 3.5  on Windows 7 doesn't support `TLS 1.2`.

Error on `make virtualbox/eval-win7x86-enterprise` : 
```
Exception calling "DownloadString" with "1" argument(s): "The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel."
```

The same download of wget with `System.Net.WebClient` on an updated Windows 7 SP1 + .Net 4.7.1 works fine. 

See this KB regarding the support of TLS 1.2 on .NET 3.5.1: https://support.microsoft.com/en-us/help/3154518/support-for-tls-system-default-versions-included-in-the-net-framework

**Comment by arizvisa** [comment]
@2xyo, do you mind if I close this and mark it as a duplicate of #177?

Despite this being a bug, it'll likely be part of the fix for #177.

**Comment by 2xyo** [comment]
Ok thanks :)
